plugin/pkg/admission/limitranger/admission.go

```diff
@@ -466,6 +469,14 @@ func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) b
 	if a.GetKind().GroupKind() == api.Kind("Pod") && a.GetOperation() == admission.Update {
 		return false
 	}
+	if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) {
```

Comment:
You didn't remove the existing check. I don't see how this newly added check works.  Pod.update will return false even before hit your new check, right?

Reply:
good catch. i rebased yesterday and that action somehow lost my commit, so i applied the saved patch to the rebase, and it was rejected - i messed up copying the check but not removing it. i'll fix it.

Reply:
fixed and tested in latest commit.